src/OpenTelemetry/Trace/Tracer.cs

```diff
@@ -25,16 +25,29 @@ namespace OpenTelemetry.Trace
     public sealed class Tracer : TracerBase
     {
         private readonly SpanBuilderOptions spanBuilderOptions;
-        private readonly IExportComponent exportComponent;
         private readonly IBinaryFormat binaryFormat;
         private readonly ITextFormat textFormat;
 
-        public Tracer(IRandomGenerator randomGenerator, IStartEndHandler startEndHandler, ITraceConfig traceConfig, IExportComponent exportComponent)
-            : this(randomGenerator, startEndHandler, traceConfig, exportComponent, null, null)
```

Comment:
if it does not get export component, it should probably get exporter instead and use it in `RecordSpanData`?

Reply:
Great point. Have updated implementation to include exporter.